pyvista/plotting/widgets.py

```diff
@@ -1277,56 +1272,66 @@ def add_sphere_widget(self, callback, center=(0, 0, 0), radius=0.5,
         Parameters
         ----------
         callback : callable
-            The function to call back when the widget is modified. It takes a
-            single argument: the center of the sphere as a XYZ coordinate.
+            The function to call back when the widget is modified. It
+            takes a single argument: the center of the sphere as a XYZ
+            coordinate.
 
         center : tuple(float)
-            Length 3 array for the XYZ coordinate of the sphere's center
-            when placing it in the scene. If more than one location is passed,
-            then that many widgets will be added and the callback will also
-            be passed the integer index of that widget.
+            Length 3 array for the XYZ coordinate of the sphere's
+            center when placing it in the scene. If more than one
+            location is passed, then that many widgets will be added
+            and the callback will also be passed the integer index of
+            that widget.
```

Comment:
If this is true then this signature should be clarified in the description of the `callback` parameter.

Reply:
Done.